### PR TITLE
Created Two Ball Auto Command

### PR DESCRIPTION
**Pull request by eddiert1239**

### CODE REVIEW

File: src/main/java/frc/team2412/robot/commands/autonomous/TwoBallAutoCommandLeft.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import frc.team2412.robot.subsystem.ShooterSubsystem;
+import frc.team2412.robot.subsystem.ShooterVisionSubsystem;
+
+public class TwoBallAutoCommandLeft extends SequentialCommandGroup {
+    public TwoBallAutoCommandLeft(IndexSubsystem indexSubsystem, ShooterSubsystem shooterSubsystem,
+            ShooterVisionSubsystem shooterVisionSubsystem, DrivebaseSubsystem drivebaseSubsystem,
+            IntakeSubsystem intakeSubsystem) {
+        Trajectory robotPath = new Trajectory(
+                new SimplePathBuilder(new Vector2(337, 133), Rotation2.fromDegrees(-26.4))
+                        .lineTo(new Vector2(449, 84), Rotation2.fromDegrees(26.7))
```
Comment (bobtseattle) [inline src/main/java/frc/team2412/robot/commands/autonomous/TwoBallAutoCommandLeft.java:29]: How far past the second ball does the robot go before stopping and shooting? (Does it make sure it goes far enough to pick the ball up?)
Is the angle of the robot at the end the best for shooting or the beginning of teleop?

**Review by bobtseattle — COMMENTED** [review]
Please get some comments to make sure there is no confusion.

File: src/main/java/frc/team2412/robot/commands/autonomous/TwoBallAutoCommandLeft.java (hunk 1 above)
Comment (eddiert1239) [reply src/main/java/frc/team2412/robot/commands/autonomous/TwoBallAutoCommandLeft.java:29]: the robot is set to go essentially on top of the ball. The angle at the end has the turret facing toward the hub, and the robot facing away from it.

**Review by jamesdooley4 — APPROVED** [review]

File: src/main/java/frc/team2412/robot/commands/autonomous/TwoBallAutoCommandLeft.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        addCommands(
+                new ParallelCommandGroup(
+                        new ScheduleCommand(new ShooterTargetCommand(shooterSubsystem, shooterVisionSubsystem)),
+                        new WaitCommand(1)),
+                new ParallelDeadlineGroup(new WaitCommand(1), new IndexShootCommand(indexSubsystem)),
+                new IntakeExtendCommand(intakeSubsystem),
+                new ParallelCommandGroup(
+                        new Follow2910TrajectoryCommand(drivebaseSubsystem, robotPath),
+                        new IntakeInCommand(indexSubsystem, intakeSubsystem)));
+
```
Comment (jamesdooley4) [inline src/main/java/frc/team2412/robot/commands/autonomous/TwoBallAutoCommandLeft.java:46]: Consider adding a wait after picking up the ball, as the command & motors will all stop as soon as the trajectory completes.

Comment (jamesdooley4) [reply src/main/java/frc/team2412/robot/commands/autonomous/TwoBallAutoCommandLeft.java:46]: Ditto for the other two commands below.

Comment (jbko6) [reply src/main/java/frc/team2412/robot/commands/autonomous/TwoBallAutoCommandLeft.java:46]: IntakeInCommand never finished, so this wouldn't really do much.

**Pull request merged**